src/Contract/PlutusData.purs

```diff
@@ -106,8 +109,17 @@ getDatumByHash
 getDatumByHash = wrapContract <<< QueryM.getDatumByHash
 
 -- | Get `PlutusData`s given a an `Array` of `DataHash`.
+-- | This function discards all possible error getting a `DataHash`.
 getDatumsByHashes
   :: forall (r :: Row Type)
    . Array DataHash
   -> Contract r (Map DataHash Datum.Datum)
 getDatumsByHashes = wrapContract <<< QueryM.getDatumsByHashes
+
+-- | Get `PlutusData`s given a an `Array` of `DataHash`.
+-- | In case of error, the returned string contains the needed information.
+getDatumsByHashes'
```

Comment:
For completeness, lets also replicate the test in `Test.AffInterface.testOgmiosDatumCacheGetDatumsByHashes` for this call

Reply:
And `Test.Plutip`'s `runPlutipContract: Datums`

Reply:
I going to add `Test.AffInterface.testOgmiosDatumCacheGetDatumsByHashes'` but those checks are more or less pointless right now since the `hash` used in the tests is outdated and all the test returns the same edge case of not datums founds.